src/main/java/run/halo/sticker/StickerPlugin.java

```diff
@@ -1,14 +1,30 @@
 package run.halo.sticker;
 
 import org.springframework.stereotype.Component;
+import run.halo.app.extension.SchemeManager;
 import run.halo.app.plugin.BasePlugin;
 import run.halo.app.plugin.PluginContext;
+import run.halo.sticker.model.Sticker;
+import run.halo.sticker.model.StickerGroup;
 
 @Component
 public class StickerPlugin extends BasePlugin {
+    private final SchemeManager schemeManager;
 
-    public StickerPlugin(PluginContext pluginContext) {
+    public StickerPlugin(PluginContext pluginContext, SchemeManager schemeManager) {
         super(pluginContext);
+        this.schemeManager = schemeManager;
     }
 
+    @Override
+    public void start() {
+        schemeManager.register(Sticker.class);
+        schemeManager.register(StickerGroup.class);
+    }
+
+    @Override
+    public void stop() {
+        schemeManager.unregister(schemeManager.get(Sticker.class));
+        schemeManager.unregister(schemeManager.get(StickerGroup.class));
```

Comment:
建议使用如下写法，防止插件启动失败后产生无法找到资源的错误。

```suggestion
        schemeManager.unregister(Scheme.buildFromType(Sticker.class));
        schemeManager.unregister(Scheme.buildFromType(StickerGroup.class));
```